scripts/validate-android-test-env.sh

```diff
@@ -10,7 +10,7 @@
 # Check that Buck is working.
 if [ -z "$(which buck)" ]; then
   echo "You need to install Buck."
-  echo "See https://buckbuild.com/setup/install.htm for instructions."
+  echo "See https://buckbuild.com/setup/install.html for instructions."
```

Comment:
There are [multiple Buck installation links](https://github.com/facebook/react-native/search?utf8=✓&q=buckbuild.com%2Fsetup&type=) in the docs and they all point to https://buckbuild.com/setup/install.html. Maybe it's easier to make sure the link doesn't break in the future, if they're all the same? We could also change them all to https://buckbuild.com/setup/getting_started.html, which is the URL `/setup/install.html` currently redirects to, but I'm not sure if that's worth the trouble?